angular-workspace/projects/ni/nimble-angular/pipes/nimble-pipes.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
```

Comment:
Wanna make the pipe standalone instead?
https://v14.angular.io/api/core/Pipe#standalone

Reply:
At some point, maybe when we move to secondary entrypoints for everything we can consistently move everything to standalone too. That's essentially captured here: https://github.com/ni/nimble/issues/172